config/ModulesMapping.jsonc

```diff
@@ -1,40 +1,40 @@
 {
-    "Applications": "^applicationTemplates\\.|^applications\\.|^servicePrincipals\\.|^onPremisesPublishingProfiles\\.|^users.appRoleAssignment$|^groups.appRoleAssignment$",
+   //"Applications": "^applicationTemplates\\.|^applications\\.|^servicePrincipals\\.|^onPremisesPublishingProfiles\\.|^users.appRoleAssignment$|^groups.appRoleAssignment$",
```

Comment:
Let's uncomment these. We want to generate all modules,

Reply:
Was for experimental purposes. Will be removed.